graders/python/Dockerfile

```diff
@@ -4,6 +4,8 @@ FROM centos:7
 ENV PYTHONIOENCODING=UTF-8
 
 COPY requirements.txt /
+COPY autograder /python_autograder
```

Comment:
Could we call the directory *before* copy `python_autograder` as well? I find it simplifies things if names don't change 😄 (unless there is a good reason, of course).